csharp/src/Google.Protobuf.Benchmarks/WriteRawPrimitivesBenchmark.cs

```diff
@@ -44,23 +44,23 @@ namespace Google.Protobuf.Benchmarks
     public class WriteRawPrimitivesBenchmark
     {
         // key is the encodedSize of varint values
-        Dictionary<int, uint[]> varint32Values;
-        Dictionary<int, ulong[]> varint64Values;
+        Dictionary<int, uint[]> varint32Values = new();
```

Comment:
Ditto possible comment. And can we make everything explicitly private at the same time?